README.md

```diff
@@ -50,12 +53,17 @@ Install this project with:
 ```sh
     cd your-desired-folder
     git clone https://github.com/henry-dura/To-Do-List.git
+    - Open it in the Code editor (Visual studio code, sublime text e.t.c) installed on your computer.
+    - In the terminal, run npm start to start the project
```

Comment:
**[Optional]**

- [ ] It's great that you have added the **npm start** command :blush: but I guess you need to install dependencies using the **npm install** command :thinking: so it should come before **npm start** command :+1: